src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -613,6 +613,8 @@ export const selectPortfolioAccountIdsByAssetId = createCachedSelector(
   selectPortfolioAccountIds,
   selectAssetIdParamFromFilter,
   (accountIds, assetId): AccountId[] => {
+    // early return for scenarios where assetId is not available yet
+    if (!assetId) return []
```

Comment:
Can we add more context to this as a PR comment?

Reply:
Sounds good to me, gave it a try in ts-node REPL and confirmed `fromAssetId('')` will throw.

<img width="560" alt="image" src="https://user-images.githubusercontent.com/17035424/188629888-dd9360f6-233b-41e6-8f04-f23a55011d59.png">